src/content/careers.md

```diff
@@ -11,9 +11,9 @@ quotes:
     author: 'Jen Harris, User Experience Designer'
   }
   - {
-    image: './team-members/images/gerardo-gonzalez.jpg',
-    text: 'You can carve a path in the company where a true sense of ownership is felt.',
-    author: 'Gerardo Gonzalez, Engineer'
+    image: './team-members/images/marlena-medford.jpg',
+    text: 'It's gratifying to work with smart, compassionate people who are positively impacting millions of Americans every day.',
```

Comment:
Remove the outer single quote and replace with double quotes, so when using a word that has an apostrphe can be used